plugin/storage/badger/spanstore/rw_internal_test.go

```diff
@@ -215,6 +199,7 @@ func TestMergeJoin(t *testing.T) {
 	assert.Equal(uint32(2), binary.BigEndian.Uint32(merged[1]))
 }
 
+/*
```

Comment:
Is this test being re-enabled before the PR is merged?

Reply:
Ah no. I'll remove the commented test. The order is different depending if we're fetching the last index or previous ones.